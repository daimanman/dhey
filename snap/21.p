{"C":10,"Z":"3s","N":10,"Remark":"TEST TASK","Url":"http://www.baidu.com","Method":"post","TaskId":21,"Type":"","FileId":"4","Status":0,"Err":"","QH":{}}