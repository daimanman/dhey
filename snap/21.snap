{"AvgTotal":30.99753109999999,"Fastest":0.0140374,"Slowest":1.1741254,"Average":0.27431443451327425,"Rps":33.58144074425507,"AvgConn":0.0010646699115044247,"AvgDNS":0.0005151345132743362,"AvgReq":0.00013292212389380532,"AvgRes":0.22276015398230095,"AvgDelay":0.050072792035398196,"ConnMax":0,"ConnMin":0.0060218,"DnsMax":0,"DnsMin":0.0060218,"ReqMin":0.0060197,"ReqMax":0,"ResMin":0.0060197,"ResMax":0.0100256,"DelayMax":0.0040118,"DelayMin":0.3228575,"Lats":[0.4110933,0.4171082,0.4321484,0.4822818,0.4862925,0.517375,0.5183779,0.1193173,0.5725215,0.1514026,0.1323659,0.241642,0.658751,0.1092906,0.2496641,0.7038712,0.0451202,0.0541437,0.1905068,0.1183581,0.1644386,0.1193468,0.04009,0.1604494,0.1804809,0.3649721,0.1163098,0.3138351,0.1634084,0.3950372,0.4461873,0.2245971,0.0471248,0.6838193,0.0671786,0.0501337,0.0140374,0.4100632,0.2657068,0.294786,0.1443841,0.1935147,0.1473923,0.0882348,0.4030724,0.2907795,0.1664428,0.6938456,0.207552,0.2516697,0.7790503,0.0370989,0.32687,0.0852189,0.0611643,0.0822205,0.5063455,0.1604272,0.667777,0.1714542,0.080213,0.2115893,0.0671822,0.2306131,0.9816103,0.2406465,0.1504002,0.082211,0.2035679,0.089239,0.0681563,0.2516431,0.2937843,0.0360898,0.5013341,0.3389039,0.1133009,0.2095562,0.0431145,0.1764695,0.7870942,0.0671786,0.0922454,0.1373653,0.4682456,0.933483,0.3098219,0.0751999,0.0972586,0.2175787,0.5344198,0.1353603,0.2245975,0.8552814,0.0932479,0.1544112,0.0621596,0.0210558,0.0882348,0.3780059,0.3278725,0.0601612,0.2346242,0.0631679,0.1163167,0.1283418,0.2566902,0.4231549,0.2025392,0.2115634,0.5344238,0.5645301,1.1741254],"ConnLats":[0.0060218,0.0060218,0.0060218,0.0060218,0.0060218,0.0060218,0.0050179,0,0.0050179,0,0,0,0.0060218,0,0,0.0060218,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"DnsLats":[0.0060218,0.0060218,0.0060218,0.0060218,0.0060218,0.0060218,0.0050179,0,0.0050179,0,0,0,0.0060218,0,0,0.0060218,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ReqLats":[0.0010029,0,0.0010029,0,0.0010018,0,0,0.0060197,0,0,0,0.0010032,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.0009769,0,0,0,0,0,0,0,0,0,0.0010032,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.0010039,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.0010028,0.0010029,0,0,0,0,0,0,0,0,0,0,0],"ResLats":[0.1965235,0.2005138,0.211563,0.2647039,0.2727252,0.3066766,0.3057978,0.1002664,0.3489272,0.1423788,0.0982752,0.2185815,0.4361599,0.0972586,0.2245972,0.4852904,0.0340871,0.0420996,0.1644374,0.0983048,0.1504013,0.0521682,0.0341104,0.1293662,0.1744649,0.3509344,0.1092724,0.1092712,0.1413756,0.1885,0.4221233,0.199531,0.0400921,0.6727901,0.0561494,0.0431119,0.0100256,0.1834895,0.2546776,0.1935172,0.1373653,0.1874969,0.1213226,0.0762028,0.1935143,0.274736,0.1534083,0.6848221,0.1944906,0.025065,0.6918399,0.0260698,0.2556804,0.0370985,0.0491327,0.0701885,0.4953167,0.1283415,0.6487265,0.0922312,0.0691842,0.1865181,0.0571402,0.1674445,0.6587528,0.2306209,0.1403717,0.0741977,0.1945426,0.0581565,0.0601583,0.2306117,0.2867695,0.028075,0.4923103,0.3218412,0.1042774,0.2025388,0.0370985,0.1664384,0.7780704,0.0591576,0.0842241,0.1283258,0.2626985,0.919446,0.3018028,0.0661753,0.0902397,0.2095573,0.4371586,0.1263361,0.2175787,0.5434516,0.0852265,0.1413763,0.0541426,0.0130345,0.0782081,0.3699842,0.2997975,0.0521399,0.2266029,0.0501319,0.0952616,0.117312,0.2316237,0.416136,0.1895047,0.2005317,0.5264025,0.5585137,1.1671065],"DelayLats":[0.2005328,0.2025574,0.2065484,0.2035409,0.1965236,0.1946565,0.1965382,0.0130312,0.2095573,0.0090238,0.0340907,0.0220573,0.2085541,0.012032,0.0250669,0.2045438,0.0110331,0.0120441,0.0260694,0.0200533,0.0140373,0.0671786,0.0059796,0.0310832,0.006016,0.0140377,0.0070374,0.2045639,0.0210559,0.2065372,0.024064,0.0250661,0.0070327,0.0110292,0.0110292,0.0070218,0.0040118,0.2255934,0.010026,0.1012688,0.0070188,0.0060178,0.0260697,0.012032,0.2095581,0.015041,0.0130345,0.0080144,0.0130614,0.2266047,0.0872104,0.0110291,0.0711896,0.0481204,0.0120316,0.012032,0.0110288,0.0320857,0.0190505,0.079223,0.0110288,0.0250712,0.010042,0.0631686,0.3228575,0.0100256,0.0100285,0.0080133,0.0090253,0.0310825,0.007998,0.0210314,0.0060113,0.0080148,0.0090238,0.0170627,0.0080192,0.0060135,0.006016,0.0100311,0.0090238,0.008021,0.0080213,0.0090395,0.2055471,0.014037,0.0070155,0.0090246,0.0070189,0.0080214,0.0972612,0.0090242,0.0070188,0.3118298,0.0080214,0.0130349,0.008017,0.0080213,0.0090238,0.0080217,0.0270722,0.0070184,0.0080213,0.013036,0.0210551,0.0110298,0.0250665,0.006016,0.0130345,0.0110317,0.0080213,0.0060164,0.0070189],"Total":3364953900,"ErrorDist":{},"StatusCodeDist":{"200":113},"SizeTotal":31855826,"SizeReq":281909,"NumRes":113,"LatencyDistribution":[{"Percentage":10,"Latency":0.0621596},{"Percentage":25,"Latency":0.1092906},{"Percentage":50,"Latency":0.207552},{"Percentage":75,"Latency":0.4100632},{"Percentage":90,"Latency":0.658751},{"Percentage":95,"Latency":0.7870942},{"Percentage":99,"Latency":1.1741254}],"Histogram":[{"Mark":0.0140374,"Count":1,"Frequentcy":0.008849557522123894},{"Mark":0.1300462,"Count":36,"Frequentcy":0.3185840707964602},{"Mark":0.246055,"Count":31,"Frequentcy":0.2743362831858407},{"Mark":0.36206379999999994,"Count":13,"Frequentcy":0.11504424778761062},{"Mark":0.47807259999999996,"Count":11,"Frequentcy":0.09734513274336283},{"Mark":0.5940814,"Count":10,"Frequentcy":0.08849557522123894},{"Mark":0.7100901999999999,"Count":5,"Frequentcy":0.04424778761061947},{"Mark":0.8260989999999999,"Count":2,"Frequentcy":0.017699115044247787},{"Mark":0.9421077999999999,"Count":2,"Frequentcy":0.017699115044247787},{"Mark":1.0581166,"Count":1,"Frequentcy":0.008849557522123894},{"Mark":1.1741254,"Count":1,"Frequentcy":0.008849557522123894}],"TaskId":21,"Duration":3.3649539}